src/View/Helper/HtmlHelper.php

```diff
@@ -72,29 +112,4 @@ public function icon($name, array $options = [])
             'attrs' => $this->templater()->formatAttributes($options, ['tag', 'iconSet']),
         ]);
     }
-
-    /**
-     * Returns Bootstrap label markup. By default, uses `<SPAN>`.
-     *
-     * @param string $text Text to show in label.
-     * @param array|string $options Additional HTML attributes.
-     * @return string HTML icon markup.
-     */
-    public function label($text, $options = [])
```

Comment:
There's no alternative in BS4 to generate labels?  FOC/crud-view plugin using this method so I would prefer if this wasn't removed.

Reply:
https://getbootstrap.com/docs/4.0/migration/#labels-and-badges
So it seems "badge" and "label" have been merged. For now be can just keep `HtmlHelper::label()` as alias for `HtmHelper::badge()` for easy migration.